definitions/infra-unix_host/definition.yml

```diff
@@ -0,0 +1,11 @@
+domain: INFRA
```

Comment:
```suggestion
domain: EXT
```

We'll have to change the directory to reflect this too (I can't do it since the PR is in your fork @thezackm)